packages/server/README.md

```diff
@@ -161,6 +161,62 @@ class MyLogger implements Logger {
 OpenFeature.setLogger(new MyLogger());
 ```
 
+### Named clients:
+
+You can have several clients, that can be referenced by a name.
+Every client can have a different provider assigned. If no provider is assigned to a named client, the global default
+provider is used.
+
+```typescript
+import { OpenFeature, ProviderEvents } from '@openfeature/web-sdk';
+
+OpenFeature.setProvider(new YourProviderOfChoice())
+OpenFeature.setProvider("client-1", new YourOtherProviderOfChoice())
+
+// Uses YourProviderOfChoice (the default)
+const unnamedClient = OpenFeature.getClient()
+
+// Uses YourOtherProviderOfChoice as it is set explicitly
+const client1 = OpenFeature.getClient("client-1")
+
+// Uses YourProviderOfChoice as no provider is set
+const client2 = OpenFeature.getClient("client-2")
```

Comment:
Can anyone think of a situation were you would like to `close`/`shutdown` a client e.g. only `client2`? If I follow everything correctly at the moment is everything or nothing. You can not only `dispose`/`shutdown` provider2.

Reply:
Good question, also shortly thought about it.
The spec at least only defines the shutdown for API and provider🤔

Reply:
I can't think of a realistic scenario where someone would want to do this. I think it's best to leave that out until one is identified.